### ISSUE TITLE
Version parser trips over dashes, eg 22.0-ALPHA1

### ISSUE THREAD

**Issue by HiassofT** [emilsvennesson/script.module.inputstreamhelper#569]
Using the current inputstreamhelper master version (e5bc154525bcf1dea11138c30f602f441a361d79) with kodi master results in an "Unsupported Kodi version" error message.

It looks a lot like the version parser implemented in #561 can't cope with dashes in the version string:

```
2024-09-20 15:53:37.360 T:1884    error <general>: [script.module.inputstreamhelper] Version string 22.0-ALPHA1 can't be interpreted! Contains non-numerics.
2024-09-20 15:53:37.360 T:1884  critical <general>: [script.module.inputstreamhelper] Unsupported Kodi version for Widevine: 22.0-ALPHA1
```

Full kodi debug log is here: https://paste.libreelec.tv/divine-sheepdog.log

**Comment by emilsvennesson** [comment]
Can you test if https://github.com/emilsvennesson/script.module.inputstreamhelper/pull/571 solves it?

**Comment by emilsvennesson** [comment]
Fixed on 0.7.0.
